src/test/regress/expected/multi_name_lengths.out

```diff
@@ -125,12 +125,69 @@ SELECT "Constraint", "Definition" FROM table_checks WHERE relid='public.name_len
 (1 row)
 
 \c - - :master_host :master_port
--- Placeholders for RENAME operations
-\set VERBOSITY TERSE
+-- Rename the table to a too-long name
+SET client_min_messages TO DEBUG1;
+SET citus.force_max_query_parallelization TO ON;
+ALTER TABLE name_lengths RENAME TO name_len_12345678901234567890123456789012345678901234567890;
+DEBUG:  the name of the shard (name_len_12345678901234567890123456789012345678_fcd8ab6f_xxxxx) for relation (name_len_12345678901234567890123456789012345678901234567890) is too long, switching to sequential and local execution mode to prevent self deadlocks
+ALTER TABLE name_len_12345678901234567890123456789012345678901234567890 RENAME TO name_lengths;
```

Comment:
I think it might be good to have `SELECT * FROM ....` so that we make sure that all the shards are in a good shape as well

Reply:
done